client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -32,6 +32,11 @@ const DataFilesSearchbar = ({ api, scheme, system, className }) => {
     history.push(`/workbench/data/${api}/${scheme}/${system}/${qs}`);
   };
 
+  // Reset form field on route change
+  useEffect(() => {
+    !hasQuery && setQuery('');
+  }, [hasQuery]);
```

Comment:
I think pathname (  `const { pathname } = useLocation();`) might be needed here as well as if user doesn't submit the search, the search bar still has the typed value when changing systems.
```suggestion
  }, [hasQuery, pathname]);
```

Reply:
I think keeping the query if the user *doesn't* submit a search is fine, this is more about indicating to the user that they're no longer looking at search results when they navigate away.

Reply:
Sounds good.  I'm fine either way.  👍  @rstijerina , can you confirm the original ticket what is needed?

Reply:
Ah, I think we do want the search field cleared if the user navigates away, even if the search was not submitted

Reply:
I have asked in [#cep-design](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1614966670038400) for direction.

Reply:
Confirmed. Please achieve Sal's described UX.

>[The search field] should be cleared if [the user navigates away]. Regardless of whether you submitted or not. It is common that navigation resets things."
> — [source](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1615225676040100?thread_ts=1614966670.038400&cid=CQUA4D5KJ)